crates/iceberg/src/scan.rs

```diff
@@ -178,9 +241,16 @@ pub struct FileScanTask {
 pub type ArrowRecordBatchStream = BoxStream<'static, crate::Result<RecordBatch>>;
 
 impl FileScanTask {
-    /// Returns a stream of arrow record batches.
-    pub async fn execute(&self) -> crate::Result<ArrowRecordBatchStream> {
-        todo!()
+    pub fn data_file(&self) -> ManifestEntryRef {
+        self.data_file.clone()
+    }
+
+    pub fn start(&self) -> u64 {
+        self.start
+    }
+
+    pub fn length(&self) -> u64 {
+        self.length
```

Comment:
Currently these fields are only used to by `open` methods, so I think it's unnecessary to expose them?